assembl/static2/js/app/components/common/avatarImage.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const AvatarImage = ({ userId, userName }) => {
+  const src = userId ? `/user/id/${userId}/avatar/30` : '/static2/img/icons/avatar.png';
```

Comment:
we should use a route here, no ?

Reply:
I don't really know, I just refactored a component to split it into a component and a subcomponent. I kept its contents as much as possible, to not break anything.